theme.js

```diff
@@ -0,0 +1,41 @@
+//Prepares the document (check for jQuery). Remember to load jQuery in the html file BEFORE running any scripts with jQuery.
+$(document).ready(function () {
+
+    // Get the first header iframe element with the id of header
+    var header = $('#header');
```

Comment:
Remember how I said I now assigned ids to the header and footer, with their same names respectively? This is why, so you can assign it to ``var header``, and selecting it using the ``$()`` syntax. Notice that it follows css selectors such as . for classes and # for ids.

Reply:
Interesting! It's much easier to follow.